bve-build/src/main.rs

```diff
@@ -168,31 +186,72 @@ fn build_shaders() {
                 break;
             };
 
-            let spirv_name = name.replace(".glsl", ".spv");
-            let out_path = entry.path().parent().expect("Must have parent").join(&spirv_name);
-
-            println!("Compiling {} to {}", name, spirv_name);
-
-            let mut child = Command::new("glslc")
-                .args(&[
-                    "-x",
-                    "glsl",
-                    stage,
-                    &format!("{}", entry.path().display()),
-                    "-o",
-                    &format!("{}", out_path.display()),
-                ])
-                .spawn()
-                .expect("Unable to find glslc in PATH. glslc must be installed. See https://github.com/google/shaderc");
-
-            let result = child.wait().expect("Unable to wait for child");
-            if !result.success() {
-                println!(
-                    "glslc failed on file {} with error code {}",
-                    name,
-                    result.code().expect("Unable to get error code")
-                );
-                exit(1);
+            {
+                let spirv_name = name.replace(".glsl", ".spv");
+                let out_path = entry.path().parent().expect("Must have parent").join(&spirv_name);
+
+                if out_of_date(entry.path(), &out_path) {
+                    println!("Compiling {} to {}", name, spirv_name);
+
+                    let mut child = Command::new("glslc")
+                        .args(&[
+                            "-x",
+                            "glsl",
+                            "-g",
+                            stage,
+                            &format!("{}", entry.path().display()),
+                            "-o",
+                            &format!("{}", out_path.display()),
+                        ])
+                        .spawn()
+                        .expect(
+                            "Unable to find glslc in PATH. glslc must be installed. See https://github.com/google/shaderc",
```

Comment:
Document this build requirement somewhere useful

Reply:
It's in the help message